src/vue/pages/Dashboard.vue

```diff
@@ -126,7 +126,12 @@ export default {
       vuexTypes.isAccountCorporate,
       vuexTypes.accountBalances,
       vuexTypes.defaultQuoteAsset,
+      vuexTypes.transferableBalancesAssets,
     ]),
+    isTransferable () {
```

Comment:
remove.
An asset has policies, by which the ability to check permission.
If you look on "Asset" class, you can see method isTransferable